### DISCUSSION TITLE
Remove or fix Alacrity cask on macOS

### DISCUSSION THREAD

**Discussion by sweenish** [Homebrew/discussions#3256] [Casks]
As it stands, I cannot run Alacritty without disabling security features, which I won't do.

A default `brew install alacritty` is broken on macOS, with less than zero interest from the devs to fix it: https://github.com/alacritty/alacritty/issues/4673

brew needs to either remove this cask or install it in such a way that it will work (probably with big flashing lights describing what it has to do so it will work). Given that the latter is likely not desirable, I would vote to just remove it.

**Comment by carlocab** [comment]
It seems weird to delete a cask based on what you personally are or aren’t willing to do.

The cask is installed by thousands of users with almost no complaints made towards Homebrew Cask, so I don’t think deleting it is a good option.

As with any open source project, fixes in the form of pull requests are always welcome.

**Reply by sweenish** [reply]
It seems weird to allow a cask that doesn't work out of the box, nor with any instructions on what should be done. Generally, if I install something via brew, I expect to then be able to run it. 

That's not a high bar. 

I'm not a stranger to doing things myself, but I use brew specifically because there are things I don't want to have to do myself. Alacritty stands in stark contrast to every other thing I've got installed from brew. I don't think my request is 'out there.'

**Reply by sweenish** [reply]
I'll also throw out that your last line, while technically true, is a garbage response. If I could quickly submit a fix, I would. I can't, so here I am. That attitude only chases people away.

I've already decided that I don't need to use Alacritty. That was an easy decision. I started a discussion because I came across something that isn't working as intended and wanted to put awareness out there. Not because I want to learn all about homebrew and devise a fix myself. My plate is full. I am hoping that the people who have homebrew on their plate already will take *some* kind of action.

You don't want to delete it? Fine. Then don't. If you don't want to fix it, either, just say so.

**Reply by carlocab** [reply]
Well, you’re not the only one with a full plate.

As for fixing it: that would be good, except 1) see above, 2) I can’t actually reproduce the behaviour you’re having issues with, so I still don’t know what there is to fix.

**Reply by SMillerDev** [reply]
> Generally, if I install something via brew, I expect to then be able to run it.

And generally you'd be correct. Sometimes you need a little more effort though.

> That's not a high bar.

That's a very high bar for packages where homebrew only controls the download process like in casks.

> If I could quickly submit a fix, I would. I can't, so here I am. That attitude only chases people away.

Since those people are people demanding work, I'm fine with that.

>  I came across something that isn't working as intended and wanted to put awareness out there. Not because I want to learn all about homebrew and devise a fix myself. My plate is full. 

Awareness rarely fixes things. 

> You don't want to delete it? Fine. Then don't. If you don't want to fix it, either, just say so.

We can't fix it and won't delete it given the usage.

**Comment by sweenish** [comment]
I appear unable to delete this topic, so someone with the ability to do so can feel free.

**Comment by ivaquero** [comment]
Comment the `terminfo` parts of the binary section can fix the issue temporarily.

This issue is raised by wrong permission of `/use/share/terminfo/`. This cask was able to make a soft link, but currently not after the macOS upgrade.

**Comment by lukep-coxauto** [comment]
you just have to open Applications and shift+right click+open and it will give you the option to open it.  only have to do this once.  this is common for downloaded software that doesn't have the right entitlement.  Granted I agree that it would be nice if the cask install didn't require this, but it's not like this is a problem never seen on mac before (electronmail as an example also has this problem on mac).

**Reply by elijaholmos** [reply]
this is no longer possible with macos 15
